sirepo/events.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+u"""Sirepo event registration and emittance.
+
+This module handles registering callbacks for events and calling the callbacks
+when the event occurs.
+
+For example, when a user logs out the AUTH_LOGOUT event is emitted. Other
+areas of the code can register a callback for this event (ex to clear jupyterhub
+cookies so the user is logged out of jupyterhub too).
+
+:copyright: Copyright (c) 2020 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+import aenum
+
+_HANDLERS = PKDict()
```

Comment:
I don't think it makes sense to use aenum.  It's unclear how you emit events. 

Just do this and drop both `_init` and the class definition.

```py
#: Map of events to handlers. Note: this is the list of all possible events.
_MAP = PKDict(
     auth_logout=[],
     end_api_call=[],
     github_authorized=[],
)
```

Reply:
a357ebdd